documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/README.md

```diff
@@ -22,42 +22,42 @@ The **On-Page Actions** category contains test actions that simulate user activi
 
 You can add these actions in two ways:
 
-* Add them to a test and specify its parameters manually.
+* Add them to a test and specify their parameters manually.
 * [Record them](../../README.md). You can record most of the on-page actions by interacting with a tested webpage. TestCafe Studio automatically adds corresponding on-page actions with the necessary parameters to a test.
 
 ## Click
 
-Clicks a webpage element with the left mouse button.
+Click a webpage element (left mouse button).
 
 Parameter | Type  | Description
 --------- | ----- | -------------
-*[Element Selector](action-parameters.md#element-selector)* | [Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) | Identifies the webpage element being clicked.
+*[Element Selector](action-parameters.md#element-selector)* | [Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) | Identifies the clicked webpage element.
```

Comment:
@dirk-pieterse 

May be, _Identifies the webpage element to be clicked._ ? We are going to click the element, we haven't clicked it yet.

Reply:
How can it identify something that has not happened yet? It predicts that you are going to perform some action?

Reply:
You can add this action not during recording, but after it. When you add the action, you first need to specify what DOM element you need to click. It is the Element Selector. The selector indentifies the element you are going to click on the webpage.

Reply:
Specifies which web element should be clicked.